src/sentry/services/hybrid_cloud/log/impl.py

```diff
@@ -19,7 +19,7 @@ def record_audit_log(self, *, event: AuditLogEvent) -> None:
             if '"auth_user"' in error_message:
                 # It is possible that a user existed at the time of serialization but was deleted by the time of consumption
                 # in which case we follow the database's SET NULL on delete handling.
-                entry.actor_user_id = None
+                event.actor_user_id = None
```

Comment:
This is the only behavioral change. I think it's just a bug, where it always should have been `event`.

The presence of the bug suggests a gap in unit testing, in case anyone wants to look into that.